src/tls_dummy.c

```diff
@@ -3,7 +3,6 @@
 #if !MG_ENABLE_MBEDTLS && !MG_ENABLE_OPENSSL && !MG_ENABLE_CUSTOM_TLS
 void* mg_tls_ctx_init(const struct mg_tls_opts *opts) {
   (void) opts;
-  MG_ERROR(("TLS is not enabled"));
```

Comment:
This should let the user build and run with no TLS, using the same code base, without complaining